packages/app/src/app/overmind/effects/vscode/extensionHostWorker/bootstrappers/client-ext-host.ts

```diff
@@ -26,10 +27,10 @@ childProcess.addForkHandler(
 
 initializeAll().then(() => {
   // Preload the TS worker for fast init
-  childProcess.preloadWorker(
+  childProcess.preloadForkHandler(
     '/extensions/node_modules/typescript/lib/tsserver.js'
   );
 
   // eslint-disable-next-line
-  require('../workers/ext-host-worker');
+  import('../workers/ext-host-worker');
```

Comment:
Import, intereeeeesting. Then we load it async?

Reply:
import is what triggers the code splitting correctly :)